doc/data-warehouse-what-and-why.md

```diff
@@ -0,0 +1,258 @@
+
+In our domain, as per the nature of the information that we share in GOV.UK,
+some user interactions follow seasonal patterns. If on a particular week or
+month of the year a set of pages receive a spike on the number of visits, it is
+very likely that the following year, on the same period, those pages will
+receive at least, a similar number of visits.
+
+Let's explore the following example:
+
+As per the spike of visits in June last year, it is very likely that
+`Organisation A` expects a spike in page views for a content item, so they might
+consider reviewing in advance the content, so it is accurate and up to date.
+
+![predictions][figure-4-predictions]
+
+# Why a Data Warehouse
+
+Now that we have a better understanding of the needs behind tracking metrics at
+scale and through time, let's explore why we have chosen a Data Warehouse as the
+underlying technology to support this functionality.
+
+## Overview
+
+Considering then user needs, and the long term vision, it seems that what we
+need is:
+
+```
+A way to track daily snapshots of performance metrics, from multiple sources, of
+all our content over time, so we can analyze it to make informed decisions about
+content.
+```
+
+If we look into the Wikipedia definition of a Data Warehouse:
+
+```
+A central repository of integrated data from one or more disparate sources, that
+stores current and historical data in one single place that is used for creating
+analytical reports for workers throughout the enterprise.
+```
+
+And, if we replace:
+
+source|replaced
+-------------------------------|--------------------------
+one or more disparate sources | Feedex, Google Analytics
+workers | content designers
+the enterprise | All GOV.UK organizations
+
+We get:
+
+```text
+A central repository of integrated data from Feedex, Google Analytics, PAAS...,
+that stores current and historical data in one single place that is used for
+creating analytical reports for content designers throughout all GOV.UK
+organizations.
+```
+
+So, it seems that from a definition / formal perspective, it fits in what we
+need to track metrics through time.
+
+## Track content changes over time for Content Items
+
+We not only need to track metrics about how content is performing through time,
+but we also need to monitor the content attributes, so we understand the
+relationship between content and performance changes through time.
+
+Say, for example, that a content item was created and published three years ago.
+To get a good understanding of how the content changes are related to their
+performance, we would need to:
+
+1. Track each change to the Content Item through the three years
+2. Update the quality metrics each time a new version is created
+3. Link the performance metrics to each one of the versions of the Content Item.
+
+Tracking the history of each content item is key to understand the relationship
+between content and performance.
+
+## A new application for GOV.UK?
```

Comment:
I don't understand this heading in the context of this `Why a Data Warehouse`.

Reply:
I assume this section is about - why build a new application within GOV.UK?

Reply:
I have renamed this section to State of the art and remove a couple of paragraphs